Application/src/visuals/Renderable.java

```diff
@@ -20,4 +20,10 @@
 
     // Returns the Object's Coordinates.
     Coordinates getCoordinates();
+
+    // Returns the name for this object.
+    String getName();
+
+    // Places the object at the specified coordinates.
+	void place(Coordinates coordinates, int size);
```

Comment:
This does nothing right?

Reply:
It does in DiagramClass. This implementation is then called in Draw.

Reply:
I missed that it was an interface my bad